packages/block-editor/src/hooks/layout.js

```diff
@@ -32,6 +32,40 @@ import { getLayoutType, getLayoutTypes } from '../layouts';
 
 const layoutBlockSupportKey = '__experimentalLayout';
 
+/**
+ * Generates the utility classnames for the given blocks layout attributes.
+ *
+ * @param { Array } attributes Array of block attributes.
+ *
+ * @return { Array } Array of CSS classname strings.
+ */
+function getLayoutClasses( attributes ) {
+	const layoutClassnames = [];
+
+	if ( ! attributes.layout ) {
+		return layoutClassnames;
+	}
+
+	if ( attributes?.layout?.orientation ) {
+		layoutClassnames.push( `is-${ attributes.layout.orientation }` );
```

Comment:
@andrewserong is there a js equivalent of sanitize_title that we should use here?

Reply:
Other comparable files in this package use lodash's `kebabCase`. I usually try to avoid adding in lodash, but since this function is already used in the package and it seems like a straightforward way to do it, personally, I'd use that 🙂

https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/hooks/font-family.js#L76